pysmt/solvers/portfolio.py

```diff
@@ -0,0 +1,255 @@
+#
+# This file is part of pySMT.
+#
+#   Copyright 2014 Andrea Micheli and Marco Gario
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+import logging
+from multiprocessing import Process, Queue, Pipe
+
+from pysmt.solvers.solver import IncrementalTrackingSolver, SolverOptions
```

Comment:
Only if you believe with all your heart! :wink: 

pysmt.solver imports SolverOptions, therefore it is visible from there. I left it like this, in case we want to split the files in the future.